packages/staking-ui/src/components/Validator/ValidatorCollapseContent.tsx

```diff
@@ -21,10 +21,11 @@ import "./ValidatorCollapseContent.css";
 interface IValidatorCollapseContentProps {
   validator?: IValidator;
   refresh?: () => unknown;
+  isEmergency?: boolean;
```

Comment:
should rename to `forceActionButtonsEnabled`

Reply:
Done